src/stub/src/arch/powerpc/64le/bxx.S

```diff
@@ -37,7 +37,7 @@ ppcbxx:  # (*f_unf)(xo->buf, out_len, h.b_cto8, h.b_ftid);
 #define ptr0 a4
 
         cmplwi cr0,ftid,0xd0;       bnelr- cr0  # if (0xd0!=ftid)   return;
-        rlwinm. len,len,32-2,2,31; beqlr- cr0  # if (0==(len>>=2)) return;
+        rldic. len,len,2,1; beqlr- cr0  # if (0==(len>>=2)) return;
```

Comment:
In the pull request, the submitted line 40 "rldic. len,len,2,1" gets the wrong answer in the register.  Here's a gdb session using register r0:
```
0x000000001000052c in .main ()
=> 0x1000052c <.main+20>:	rldic.  r0,r0,2,1
(gdb) p $r0
$1 = 0xffffffffffffffff  # all bits 1
(gdb) g  ## g is a macro for "stepi\n x/i $pc"
0x0000000010000530 in .main ()
=> 0x10000530 <.main+24>:	li      r0,4095
(gdb) p $r0
$2 = 0x7ffffffffffffffc  # we want 0x3f....ff
(gdb) g
0x0000000010000534 in .main ()
=> 0x10000534 <.main+28>:	rldic.  r0,r0,2,1
(gdb) p $r0
$4 = 0xfff
(gdb) g
0x0000000010000538 in .main ()
=> 0x10000538 <<not relevant>>
(gdb) p $r0
$5 = 0x3ffc  # we want 0x3ff
`````
The proposed "rldic len,le,0,2" ["two hours ago"] has  problems.  The source register should be 'len', not 'le'.  Even after fixing that obvious typo, the rotate count is 0: bits do not move.  We want a rotate count of (64-2) so that most bits move 2 places towards the LSB, as in ">>2".  And there is no dot '.' at the end of the opcode (or other syntax) to force recording of the condition.  

Please use "rldicl. len,len,64-2,2" which is pedantically correct, and is what gdb shows.  If you want to use something else then please  provide a stepi-by-stepi proof.